conda_index/index/convert_cache.py

```diff
@@ -72,7 +72,13 @@ def create(conn):
     # has md5, shasum. older? packages do not include timestamp?
     # SELECT path, datetime(json_extract(index_json, '$.timestamp'), 'unixepoch'), index_json from index_json
     conn.execute(
-        "CREATE TABLE IF NOT EXISTS index_json (path TEXT PRIMARY KEY, index_json BLOB)"
+        """
+        CREATE TABLE IF NOT EXISTS index_json (
+            path TEXT PRIMARY KEY, index_json BLOB,
+            name AS (json_extract(index_json, '$.name')),
+            sha256 AS (json_extract(index_json, '$.sha256'))
```

Comment:
It might be worthwhile to index name; but it might not since we will typically fetch everything.